tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1397,6 +1411,22 @@ def check_parameters(self, node):
                 msgid="docstring-keyword-should-match-keyword-only", args=(", ".join(missing_kwonly_args)), node=node, confidence=None
             )
 
+        # check that all types are formatted correctly
+        for keyword, doc_type in docstring_keyword_args.items():
```

Comment:
I think it'll would be more efficient to use _list comprehension_ logic for both _for cases_.